src/ui/components/Widgets/Chart/components/WidgetHeader.tsx

```diff
@@ -61,17 +64,49 @@ export const WidgetHeader = (props: HeaderProps) => {
     const size = DL.IS_MOBILE ? MOBILE_SIZE.TABS : 'm';
 
     const widgetTitle = currentTab?.title || title;
+    const widgetTitleHint =
+        currentTab?.enableHint && currentTab?.hint?.trim() ? currentTab?.hint?.trim() : '';
     const showFiltersClear = showActionParamsFilter && onFiltersClear;
 
+    const handleClickHint = React.useCallback((e: React.MouseEvent) => {
+        e.stopPropagation();
+        return false;
+    }, []);
+
     const renderTabs = () => {
         if (DL.IS_MOBILE && (isFullscreen || (!hideTitle && tabsItems?.length === 1))) {
-            return <div className={b('mobile-title')}>{widgetTitle}</div>;
+            return (
+                <div className={b('mobile-title')}>
+                    <div className={b('mobile-title-wrap')}>
+                        {widgetTitle}
+                        {Boolean(widgetTitleHint) && (
+                            <MarkdownHelpPopover
+                                markdown={widgetTitleHint}
+                                className={b('chart-title-hint')}
+                                buttonProps={{
+                                    className: b('chart-title-hint-button'),
+                                }}
+                                {...(DL.IS_MOBILE
```

Comment:
There's `if (DL.IS_MOBILE &&` in the if block just above, so the second check of it is not necessary